.github/workflows/lint-test.yaml

```
@@ -0,0 +1,26 @@
+name: Lint
```

@szaher Shall we just name this Workflow and rename file to `test-python.yaml`:
```suggestion
name: Unit Test - Python
```

Similar to Trainer test: https://github.com/kubeflow/trainer/blob/master/.github/workflows/test-python.yaml#L14, we can just run 2 jobs: pre-commit + test